### PR TITLE
feat: Updated room 2 and 3 graphics

### PR DESCRIPTION
**Pull request by balloonpopper**
feat: Updated room 1 graphics. Merged child scenes into master scene for clarity.
Previously shared bridge resource moved to level3 folder, and button resource moved to level9 folder.

### CODE REVIEW

**Comment by StraToN** [comment]
I have tested your branch. It looks awesome! However your change broke room03 (probably because room03 references the bridge from room02), so the game crashes when you leave room02 to room03.

**Review by dploeger — CHANGES_REQUESTED** [review]
The animation is really great! Thanks!

@StraToN @BHSDuncan I'm not 100% sure about all the whitespace changes. While we should definitely apply them, my "cleanness" feeling suggests moving them to another PR to not mix things up. I'll let you decide.

**Comment by dploeger** [comment]
@balloonpopper I guess you didn't make that changes manually, right? Was it some kind of autoformat which caused them? Maybe we can run that autoformatter over the whole sourcecode in another PR.

**Comment by balloonpopper** [comment]
@dploeger  / @BHSDuncan  - The edits to the gd files are made automatically by Godot when you run the project. I've removed the whitespace in all .gd with sed under another change now - see https://github.com/godot-escoria/escoria-demo-game/pull/494

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ bin
 *.import
 .import
 *.translation
-.godot
\ No newline at end of file
+.godot
+addons/escoria-core/default_bus_layout.tres
```
[the remark this answers is not in the document]
Comment (balloonpopper) [reply .gitignore]: Removed. I'll raise it as a bug instead.

**Review by dploeger — APPROVED** [review]

**Comment by dploeger** [comment]
Wonderful. Thanks a lot! 🎁

File: addons/escoria-core/default_bus_layout.tres [hunk 2]
```diff
@@ -1,12 +1,11 @@
 [gd_resource type="AudioBusLayout" format=2]
 
 [resource]
-bus/0/volume_db = -8.89811
 bus/1/name = "SFX"
 bus/1/solo = false
 bus/1/mute = false
 bus/1/bypass_fx = false
-bus/1/volume_db = 0.0
+bus/1/volume_db = -60.0
```
[the remark this answers is not in the document]
Comment (StraToN) [reply addons/escoria-core/default_bus_layout.tres]: Yes I always do as well. Not sure what to do with them. I usually commit the change, hoping that it won't be required anymore for others.

**Review by StraToN — APPROVED** [review]

File: addons/escoria-core/default_bus_layout.tres (hunk 2 above)
[the remark this answers is not in the document]
Comment (dploeger) [reply addons/escoria-core/default_bus_layout.tres]: Recorded in #107

**Pull request merged**